AntistasiOfficial.Altis/Save/fn_saveFunctions.sqf

```diff
@@ -414,6 +414,10 @@ fn_setData = {
 				clearBackpackCargoGlobal caja;
 				{caja addBackpackCargoGlobal [_x,1]} forEach _varValue;
 			};
+			//Redundant, same code done for all options, propably add some bool to avoid double execution??
+			//Most likely getting even worse with supply levels ...
+			//I would recommend to redo this part, if I got some time, I will do this - wurzel
+			
```

Comment:
@Barbolani77 can be a good idea for WotP? too? haven't checked if you changed it or not